recaf-ui/src/main/resources/languages/bytecode.json

```diff
@@ -16,8 +16,8 @@
+			"pattern": "\\b((?:i(?:n(?:v(?:oke(?:s(?:pecial(?:interface)?|tatic(?:interface)?)|virtual(?:interface)?|interface|dynamic)|isibl(?:e-typ)?e-annotation)|stanceof|eg)|f(?:_(?:icmp(?:g[et]|l[et]|eq|ne)|acmp(?:eq|ne))|n(?:(?:onn)?ull|e)|g[et]|l[et]|eq)|a(?:(?:[dn]|loa)d|store)|const_(?:[02345]|m?1)|s(?:h[lr]|tore|ub)|m(?:plements|ul)|(?:ush|x?o)r|re(?:turn|m)|2[bcdfls]|load|div|inc)|l(?:a(?:(?:[dn]|loa)d|store|bel)|s(?:h[lr]|tore|ub)|c(?:onst_[01]|mp)|o(?:okupswitch|r)|re(?:turn|m)|(?:ush|xo)r|d(?:iv|c)|2[dfi]|load|ine|mul|neg)|d(?:up(?:2(?:_x[12])?|_x[12])?|a(?:(?:loa|d)d|store)|c(?:onst_[01]|mp[gl])|ef(?:ault|ine)|re(?:turn|m)|s(?:tore|ub)|2[fil]|load|div|mul|neg)|f(?:c(?:onst_[012]|mp[gl])|a(?:(?:loa|d)d|store)|i(?:eld|nal)|re(?:turn|m)|s(?:tore|ub)|2[dil]|_new|load|div|mul|neg)|a(?:n(?:notation|ewarray)|r(?:raylength|eturn)|a(?:store|load)|l(?:ias|oad)|const_null|bstract|store|throw)|s(?:y(?:chronized|nthetic)|i(?:gnature|push)|t(?:rictfp|atic)|a(?:store|load)|wap)|p(?:u(?:t(?:static|field)|blic)|arameter-annotation|r(?:otected|ivate)|op2?)|m(?:onitore(?:nter|xit)|ethod(?:-type)?|ultianewarray|acro)|c(?:a(?:s(?:tor)?e|load|tch)|heckcast|lass)|t(?:ype-annotation|ableswitch|hrows|ry)|b(?:a(?:store|load)|ipush|ridge)|g(?:et(?:static|field)|oto)|n(?:ew(?:array)?|ative|op)|e(?:n(?:um|d)|xpr)|va(?:rargs|lue)|ret(?:urn)?|jsr))\\b",
+			"pattern-unoptimized": "\\b(public|protected|private|static|final|sychronized|varargs|native|abstract|strictfp|synthetic|bridge|aaload|aastore|aconst_null|alias|aload|anewarray|areturn|arraylength|astore|athrow|baload|bastore|bipush|caload|castore|catch|checkcast|d2f|d2i|d2l|dadd|daload|dastore|dcmpg|dcmpl|dconst_0|dconst_1|ddiv|define|dload|dmul|dneg|drem|dreturn|dstore|dsub|dup|dup2|dup2_x1|dup2_x2|dup_x1|dup_x2|f2d|f2i|f2l|f_new|fadd|faload|fastore|fcmpg|fcmpl|fconst_0|fconst_1|fconst_2|fdiv|field|fload|fmul|fneg|frem|freturn|fstore|fsub|getfield|getstatic|goto|i2b|i2c|i2d|i2f|i2l|i2s|iadd|iaload|iand|iastore|iconst_0|iconst_1|iconst_2|iconst_3|iconst_4|iconst_5|iconst_m1|idiv|if_acmpeq|if_acmpne|if_icmpeq|if_icmpge|if_icmpgt|if_icmple|if_icmplt|if_icmpne|ifeq|ifge|ifgt|ifle|iflt|ifne|ifnonnull|ifnull|iinc|iload|imul|ineg|instanceof|invokedynamic|invokeinterface|invokespecial|invokestatic|invokevirtual|ior|irem|ireturn|ishl|ishr|istore|isub|iushr|ixor|jsr|l2d|l2f|l2i|label|ladd|laload|land|lastore|lcmp|lconst_0|lconst_1|ldc|ldiv|line|lload|lmul|lneg|lookupswitch|lor|lrem|lreturn|lshl|lshr|lstore|lsub|lushr|lxor|monitorenter|monitorexit|multianewarray|new|newarray|nop|pop|pop2|putfield|putstatic|ret|return|saload|sastore|sipush|swap|tableswitch|throws|try|value|expr|case|default|annotation|parameter-annotation|invisible-annotation|type-annotation|invisible-type-annotation|method-type|signature|enum|implements|macro|method|field|class|end|invokestaticinterface|invokespecialinterface|invokevirtualinterface)\\b"
```

Comment:
Prefer to keep the `invoke<bla>` items next to each other in the regex.

For the optimized output I use: https://myregextester.com/ (use the optimize checkbox)